ui/src/app/cohort-search/cohort-search/cohort-search.component.tsx

```diff
@@ -184,7 +184,7 @@ export const CohortSearch = withCurrentCohortSearchContext()(class extends React
   closeSearch() {
     currentCohortSearchContextStore.next(undefined);
     // Delay hiding attributes page until sidebar is closed
-    setTimeout(() => attributesSelectionStore.next(undefined), 500);
+    global.setTimeout(() => attributesSelectionStore.next(undefined), 500);
```

Comment:
Resolved `error TS2322: Type 'number' is not assignable to type 'Timeout'`.
`setTimeout` is ambigious between `global.setTimeout()` and `window.setTimeout()`.